### ISSUE TITLE
[ Feature Req./Suggestion ] Retain mute role upon joining

### ISSUE THREAD

**Issue by Terrails** [Mantaro/MantaroBot#658]
[Mod Edit: Marking as suggestion instead]
**Before submitting I have**\
(Simply replace the space inside the brackets with an X for the things you have done)
- [x] Double checked existing Issues/Pull Requests for a similar feature.

**Describe what you would like us to add**
Mantaro should retain the mute role when a user rejoins the server and a mute has not yet expired.

**Additional context**
N/A
[/Mod Edit]

<details>
<summary>Original Content</summary>

**Before submitting I have:**
- [X] Made sure this is actual unintended behaviour.
- [X] Double checked the [FAQ](https://github.com/Mantaro/MantaroBot/wiki/FAQ) for potential misunderstandings.
- [X] Double checked `~>help <command name>` (if this is command related) for potential misunderstandings.

**Describe the bug**
Basically what the title says. Tried muting a test account, left and rejoined the server and the test account wasn't muted anymore.

**To Reproduce**
Have a muted role with disabled chatting permissions and mute a user using Mantaro. When the user rejoins the server there should be no muted role on them.

**Expected behavior**
The user should get muted role when rejoining as long as the mute is still valid.

</details>

**Comment by MrLar** [comment]
This is not exactly a bug, but rather simply something that is not a feature within Mantaro, if you want me to I can mark this as a suggestion to add such a feature instead.

**Comment by Terrails** [comment]
Sure.
